source/installation-guide/installation-guide.rst

```diff
@@ -318,4 +324,5 @@ Prerequisites: kubectl installed and accesible from path
 
    4. Update the empty values in OS2IoT-docker/helm/charts/mosquitto-os2iot/values.yaml
 
-   5. Update the :code:`MQTT_BROKER_HOSTNAME` with the ip/hostname that you used for step 4 and 6 in the steps above, and :code:`CA_KEY_PASSWORD` with the password that you entered in step 3 in the steps above, in the file "OS2IoT-docker/helm/charts/os2iot-backend/deployment.yaml".
\ No newline at end of file
+   5. Update the :code:`MQTT_BROKER_HOSTNAME` with the ip/hostname that you used for step 4 and 6 in the steps above, and :code:`CA_KEY_PASSWORD` with the password that you entered in step 3 in the steps above, in the file "OS2IoT-docker/helm/charts/os2iot-backend/deployment.yaml".
```

Comment:
We should fix (my ealier mistake) with the "step 4 and 6 in the steps above". In the actual steps, step nr. 5 is missing, so it should just be 1-6 instead of 1-7